src/components/Attachments/ImageAttachment.js

```diff
@@ -2,18 +2,17 @@ import { Box } from '@rocket.chat/fuselage';
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const ImageAttachment = ({ attachment }) => {
-  return (
-    <Box>
-      <p>{attachment?.description}</p>
-      <img
-        src={'http://localhost:3000' + attachment.image_url}
-        height={attachment.image_dimensions.height}
-        width={attachment.image_dimensions.width}
-      />
-    </Box>
-  );
-};
+const ImageAttachment = ({ attachment }) => (
+  <Box>
+    <p>{attachment?.description}</p>
+    <img
+      src={`http://localhost:3000${attachment.image_url}`}
```

Comment:
Awesome work! Can we please update all these hard-coded strings. We can create a separate function called getHost inside lib/api.js RocketChat instance where we return the host.

We can get host as props for all the attachment components from the parent attachment component

Reply:
Great work 👏
Sidharth bhaiya  i also need that get host function in one of my PR 

So should i make that function or wait till this PR got merged with that function ? 

I can make that before today afternoon Easily.

Reply:
hey @VenomFate-619  i made this function in my PR you can use it from there